### PR TITLE
Issue #58: Update and Restructure Quickstart

### PR DESCRIPTION
**Pull request by heyselbi**
## Description

Addressing Issue #58 
Restructuring installation instructions
Adding user case scenario instructions

## How Has This Been Tested?

Installation instructions were tested

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: demo/kserve/deploy-remove.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+If you would like to use short scrips instead of following step-by-step manual, please see [here](#script-based-deployment-and-removal).
+
+Before proceeding, please make sure you are in the `/caikit-tgis-serving/demo/kserve/` directory.
+
+## Deploy Minio containing flan-t5-small model
+
+If you have your model in another S3-like object storage (e.g., AWS S3), you can skip this step.
+We will create a new project/namespace for minio and deploy it together with service account and data connection (secret with generated access key ID and secret access key). 
+~~~
+ACCESS_KEY_ID=THEACCESSKEY
```
Comment (vaibhavjainwiz) [inline demo/kserve/deploy-remove.md:13]: As this is just an example so we don't actually need to build ACCESS_KEY randomly. Any hardcoded key could work here as `ACCESS_KEY_ID=admin` and `SECRET_ACCESS_KEY=password`. With hardcoded value we don't need to do string manipulation and script to deploy minio would be such simpler.

**Review by vaibhavjainwiz — APPROVED** [review]
Nice work @heyselbi. 
I just have a small oberservation. Your changes LGTM.

File: demo/kserve/install-manual.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+## Install Knative Serving
+~~~
+oc create ns ${KSERVE_OPERATOR_NS}
+oc create ns knative-serving
+oc -n istio-system apply -f custom-manifests/service-mesh/smmr-${TARGET_OPERATOR_TYPE}.yaml 
+oc apply -f custom-manifests/service-mesh/peer-authentication.yaml
+oc apply -f custom-manifests/service-mesh/peer-authentication-${TARGET_OPERATOR_TYPE}.yaml 
+~~~
+This is needed because of [this](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/serverless/serving#serverless-domain-mapping-custom-tls-cert_domain-mapping-custom-tls-cert).
```
Comment (VedantMahabaleshwarkar) [inline demo/kserve/install-manual.md:47]: Nit: 
`We use PeerAuthentications to enable mTLS according to [Openshift Serverless Documentation](https://access.redhat.com/documentation/en-us/red_hat_openshift_serverless/1.28/html/serving/configuring-custom-domains-for-knative-services#serverless-domain-mapping-custom-tls-cert_domain-mapping-custom-tls-cert)`

**Review by VedantMahabaleshwarkar — APPROVED** [review]
Left a minor wording suggestion(nit)
Otherwise /lgtm

**Pull request merged**